generator/test_case_generator.go

```diff
@@ -187,6 +187,16 @@ var testTypeToTestConfig = map[string][]testConfig{
 			testDir: "./test/metric_value_benchmark",
 			targets: map[string]map[string]struct{}{"arc": {"amd64": {}}},
 		},
+		{
+			testDir:      "./test/metric_value_benchmark",
+			terraformDir: "terraform/eks/daemon/windows/2019",
+			targets:      map[string]map[string]struct{}{"arc": {"amd64": {}}},
+		},
+		{
+			testDir:      "./test/metric_value_benchmark",
+			terraformDir: "terraform/eks/daemon/windows/2022",
+			targets:      map[string]map[string]struct{}{"arc": {"amd64": {}}},
+		},
 		{
```

Comment:
Can you check if /test/emf is testing any specific feature and implement it for windows?

Reply:
Hi @nathalapooja,
these emf test are testing emf receiver/exporter in CW agent.
Since CI tests are indirectly testing emf already, this particular test can be skipped for Windows at the moment but it is still good to have this test for Windows.
I will work on adding this test for Windows as post-release activity.